server/server-dist.js

```diff
@@ -13,4 +12,4 @@ app.use(express.static("./dist"));
 app.get("*", (req, res) =>
   res.sendFile(path.resolve("dist/index.html")));
 
-app.listen(config.port.dist, () => logger.info(`Listening on http://localhost:${config.port.dist} ...`));
\ No newline at end of file
+app.listen(port, () => console.log(`Listening on http://localhost:${port} ...`));
```

Comment:
Because the logging library is only for development by now.
I didn't want to tell the user of this library which logging framework to use in production.